Resources/config/controller.xml

```diff
@@ -24,6 +24,10 @@
             <argument>%hwi_oauth.connect%</argument>
             <argument>%hwi_oauth.grant_rule%</argument>
 
+            <call method="setContainer">
+                <argument type="service" id="service_container" />
```

Comment:
as commented on the other linked PR:

but this would not be correct.

1. service keys/ids can be aliased differently in the service locator than in the public container
2. not all services are publicly available in the service container

So the only solution I see for now is to add `autowire=true` for the `LoginController`. Its not the best solution but it won't break anything.

Reply:
Oh, I missed your comment. I'll adjust this PR.